### DISCUSSION TITLE
Editor Module 0.0.1

### DISCUSSION THREAD

**Discussion by paring-chan** [paring-chan/AdofaiUtils#7] [General]
타일 비트수 세기 기능 추가
타일을 선택하면 비트 수를 표시해줍니다.
![image](https://user-images.githubusercontent.com/68010770/129017649-f991ef98-9dd2-48ab-9000-00c8bf3b1b80.png)

~~코멘트 하나씩만 달고가주세요 읍읍~~

<hr /><em>This discussion was created from the release <a href='https://github.com/pikokr/AdofaiUtils2/releases/tag/0.0.1-editor'>Editor Module 0.0.1</a>.</em>

**Comment by Upload18cute** [comment]
대충 엄지척 이모지

**Reply by paring-chan** [reply]
:thumbsup:
